charts/ocis/docs/values-desc-table.adoc

```diff
@@ -1794,6 +1794,30 @@ a| [subs=-attributes]
 a| [subs=-attributes]
 see detailed service configuration options below
 | THUMBNAILS service.
+| services.thumbnails.maintenance
+a| [subs=-attributes]
++object+
+a| [subs=-attributes]
+`{"cleanUpOldThumbnails":{"deleteBefore":30,"enabled":false,"schedule":"* * * * *"}}`
+| Maintenance configuration for cleanup jobs.
+| services.thumbnails.maintenance.cleanUpOldThumbnails.deleteBefore
+a| [subs=-attributes]
++int+
+a| [subs=-attributes]
+`30`
+| Setting that makes the command delete all thumbnails older than the value. The value is a number in days.
+| services.thumbnails.maintenance.cleanUpOldThumbnails.enabled
+a| [subs=-attributes]
++bool+
+a| [subs=-attributes]
+`false`
+| Enables a job, that cleans up old thumbnails. Requires persistence to be enabled.
```

Comment:
What is meant with:
Requires persistence to be enabled
?

Reply:
> if a pod gets deleted then data will also be deleted

ah, ok, then what about:
`Only recommended if pod persistence is setup.`

Reply:
What about that we describe what persistence means in the context of helm charts in the admin documentation?